### ISSUE TITLE
How can I change the map language? 

### ISSUE THREAD

**Issue by EyalNous** [angular-ui/angular-google-maps#1482]
Thanks.

**Comment by nmccready** [comment]
http://stackoverflow.com/questions/2662889/how-to-indicate-the-language-in-the-google-maps-api

Key being the URL:

`http://maps.google.com/maps/api/js?sensor=false&amp;language=ru-RU`

and 
`language=ru-RU`


Either use the URL above or the recommended angular approach by loading the google sdk async via below code:

```js
'use strict';
angular.module("angular-google-maps-example", ['uiGmapgoogle-maps'])
.config(['uiGmapGoogleMapApiProvider', function (GoogleMapApi) {
  GoogleMapApi.configure({
    //key: 'your api key',
    //v: '3.20', //(defaults to 3.X)
    libraries: 'weather,geometry,visualization', //optional
    language:ru-RU //set whatever langs u want
  });
}])
```

**Comment by EyalNous** [comment]
Thank you for your replay just to be sure it is uiGmapGoogleMapApi and not uiGmapGoogleMapApiProvider  right?

**Comment by nmccready** [comment]
Depends on when you use it. That nuance is due to angular design. If you use it at the config level it is `uiGmapGoogleMapApiProvider` otherwise it is `uiGmapGoogleMapApi`. This is one of the sucks of angular.

**Comment by glederer** [comment]
the language is defined in the .config of your angualr module, which isran only once.

Is there a way to change the language dynamically? suppose the map is already loaded in "EN", then I want to change it in 'FR'?

**Comment by nmccready** [comment]
Its a provider so you should be able to use it at many phases. https://gist.github.com/demisx/9605099

lastly it uses a factory internally which you have access to https://github.com/angular-ui/angular-google-maps/blob/master/src/coffee/providers/map-loader.coffee#L3


I think your main issue is you will have the google maps api loaded twice. I think the scripId stuff needs to be fixed. Fixed as to not use uuid if an elementId is passed into the options.script.id . Please feel free to make the mods and PR.

**Comment by nmccready** [comment]
Please make a new issue as this is out of scope of this ticket.
